Lombiq.Hosting.Azure.ApplicationInsights/Services/TelemetryFilter.cs

```diff
@@ -23,37 +24,25 @@ public class TelemetryFilter : ITelemetryProcessor
 
     public void Process(ITelemetry item)
     {
-        if (!ShouldSend(item)) { return; }
+        if (ShouldNotSend(item)) { return; }
 
         _next.Process(item);
     }
 
-    private static bool ShouldSend(ITelemetry item)
+    private static bool ShouldNotSend(ITelemetry item)
     {
         var dependency = item as DependencyTelemetry;
-        if (dependency is not { Success: false }) return true;
+        if (dependency is not { Success: false }) return false;
 
-        dependency.Properties.TryGetValue("Error", out var dependencyError);
-        dependency.Properties.TryGetValue("Exception", out var exception);
-        dependency.Properties.TryGetValue("OrchardCore.ShellName", out var shellName);
-        var hasError = Errors.Exists(error => dependencyError?.StartsWithOrdinalIgnoreCase(error) == true) ||
-            Errors.Exists(error => exception?.StartsWithOrdinalIgnoreCase(error) == true) ||
-            exception?.Contains(GetTenantSqlErrorMessage(shellName)) == true;
-        var shouldSend = !hasError;
-
-        if (!shouldSend)
+        if (dependency.ResultCode == "409" && dependency.Name == "BlobContainerClient.Create")
```

Comment:
Doesn't this make checking for "The specified container already exists." below unnecessary, since this is the same telemetry?

Reply:
Yeah it would, but I messed this up because I thought it sends these errors differently... Fixed it now, it still needs the additional checks.